src/main/scala/com/github/sbt/sbom/BomExtractor.scala

```diff
@@ -117,12 +116,4 @@ class BomExtractor(settings: BomExtractorParams, report: UpdateReport, log: Logg
     }
   }
 
-  private def logComponent(component: Component): Unit = {
```

Comment:
So this method is never being used which is why its deleted. We can call it somewhere to prevent it from being deleted since it seems to be useful.

@raboof wdyt?

Reply:
or maybe remove the `private`? I'm fine with either.

Reply:
Ill remove `private` then